app/src/main/proto/vss.proto

```diff
@@ -98,6 +98,79 @@ message PutObjectRequest {
 message PutObjectResponse {
 }
 
+message ListKeyVersionsRequest {
+
+  // store_id is a keyspace identifier.
```

Comment:
minor: might make sense to move repeating doc comments to the top of the file, and just link to them in each request proto.

Reply:
Yeah, i want to do that but there wasn't a clear and nice way to do it.
If we do it the current way, protobuf generated classes have nicely added field docs.

Reply:
> Yeah, i want to do that but there wasn't a clear and nice way to do it. If we do it the current way, protobuf generated classes have nicely added field docs.

Are you saying any top-level docs are not copied anywhere in the generated code? In my past experience, the generated code didn't have any docs at all, so developers would just look at the proto file. But perhaps this has changed.

We should ideally end off in a place where the reader has a high-level summary somewhere and have more detailed explanations as they drill down into messages and fields, IMO. Not sure how this will ultimately look, but I could imagine a few possibilities:
- High-level summary at top, minimal request/response docs, detailed field docs
- High-level summary at top, detailed request/response docs, minimal field docs
- No high-level summary, high-level summary in request/response docs, detailed request/response docs

Regardless of which, we should have some high-level summary somewhere so developers don't have to exhaustively read every field doc to know how everything fits together. Feel free to do in a follow-up, of couse.

Reply:
>Are you saying any top-level docs are not copied anywhere in the generated code?
Yes, they get omitted, only field and struct level docs get into generated code.

My current plan is 
"High-level summary in project docs, No high-level summary in proto, high-level summary in request/response docs, detailed field docs"

In this way, docs on top of fields and structs will get copied to generated code, and high-level project doc serves as reference for usage. It will also contain examples from client-side code.
Will take it as follow-up.